src/style.scss

```diff
@@ -111,6 +111,24 @@
   }
 }
 
+// Tune component styles
```

Comment:
We will style all tune components here this way?
There is a great example how to customize and theme component library with atomic css tailwind/unocss: https://anu-vue.netlify.app/guide/getting-started/customization.html
With creating our own component library (not just form builder and validator) comes great responsibility.

Reply:
also there is a good discussion https://github.com/onu-ui/onu-ui/discussions/126

Reply:
Tune is primarily designed for internal use within Snapshot, SX, and should not be confused with general-purpose component libraries like Vuetify. Currently, it consists mostly of fundamental components, but we will incorporating components tailored specifically to Web3 and governance applications.